src/providers/MeetingProvider/index.tsx

```diff
@@ -59,6 +59,10 @@ interface Props {
    * The [ActiveSpeakerPolicy](https://aws.github.io/amazon-chime-sdk-js/interfaces/activespeakerpolicy.html) object that you want to use in the meeting session.
    */
   activeSpeakerPolicy?: ActiveSpeakerPolicy;
+  /**
+   * Maximum amount of time in milliseconds to allow for reconnecting.
```

Comment:
Can we link to JS SDK doc like other props?

Reply:
done